src/components/header/header.tsx

```diff
@@ -47,12 +49,27 @@ function DashboardScreen () {
 
   return (
     <LoaderWrapper loading={isLoading}>
-      <div>
-        {(personTotalTime) ?`Your balance is ${getHoursAndMinutes(Number(personTotalTime?.balance))}` :null}
-      </div>
-      <button type="button" onClick={() => auth?.logout()}>Log out</button>
+      <Container sx={{ fontFamily: 'Nunito Sans' }}>
+        <Grid container sx={{
+                    "border-radius":"15px",
+                    "background-color":"#f2f2f2",
+                    "box-shadow": "5px 5px 5px 0 rgba(50,50,50,0.1)",
```

Comment:
Could this be implemented as reusable style? Since it's being used again starting from row 63 below.